relay/polkadot/src/lib.rs

```diff
@@ -130,7 +130,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("polkadot"),
 	impl_name: create_runtime_str!("parity-polkadot"),
 	authoring_version: 0,
-	spec_version: 9430,
+	spec_version: 1_000_000,
```

Comment:
@liamaharon i dont see a warning in the `try-runtime-cli` that the spec version did not increase when i use the runtime from master.  
Could you check please?

Reply:
Will check this tomorrow

Reply:
<img width="1508" alt="Screenshot 2023-10-23 at 10 21 40" src="https://github.com/polkadot-fellows/runtimes/assets/16665596/06c06f8c-310d-4b61-89e3-0c7ca2ef6ca1">

It does check and log a warning, but maybe you missed it because it logs *before* running the migrations and is easily missed.

I've created an issue to move the warning log to after the migrations execute and make it more obvious https://github.com/paritytech/try-runtime-cli/issues/45